src/main/java/lotto/domain/LottoPrice.java

```diff
@@ -0,0 +1,26 @@
+package lotto.domain;
+
+public class LottoPrice {
+    public static final int LOTTO_PRICE = 1000;
```

Comment:
다음과 같이 자릿수에 `_`를 사용하면 가독성을 높일 수 있어요.
```suggestion
    public static final int LOTTO_PRICE = 1_000;
```